mne_bids/mne_bids.py

```diff
@@ -297,6 +295,34 @@ def _participants_tsv(raw, subject_id, group, fname, overwrite=False,
     return fname
 
 
+def _participants_json(fname, overwrite=False, verbose=True):
+    """Create participants.json for non-default columns in accompanying TSV.
+
+    Parameters
+    ----------
+    fname : str
+        Filename to save the scans.tsv to.
+    overwrite : bool
+        Defaults to False.
+        Whether to overwrite the existing data in the file.
+        If there is already data for the given `fname` and overwrite is False,
+        an error will be raised.
+    verbose : bool
+        Set verbose output to true or false.
+
+    """
+    cols = OrderedDict()
+    cols['participant_id'] = {'Description': 'Unique participant identifier'}
+    cols['age'] = {'Description': 'Unique participant identifier',
```

Comment:
So I clearly did a bad job of reviewing this PR... But this line is wrong. You have the same description for and and ID.

Reply:
indeed! Fix is welcome :)

Reply:
:grimacing: thanks for catching @monkeyman192 !